lang/haxe/Makefile

```diff
@@ -25,7 +25,17 @@ USE_OCAML=		yes
 USE_OCAML_FINDLIB=	yes
 USE_OCAML_CAMLP4=	yes
 
-MAKE_JOBS_UNSAFE=yes
+MAKE_JOBS_UNSAFE=	yes
+
+post-patch:
+	${REINPLACE_CMD} 's|/usr/local|${PREFIX}/local|g' \
```

Comment:
Shouldn't that be the following?

`      ${REINPLACE_CMD} 's|/usr/local|${PREFIX}|g'` \